sdl_android/src/main/java/com/smartdevicelink/security/SdlSecurityBase.java

```diff
@@ -44,7 +44,8 @@ public void resetParams() {
     
     public void handleInitResult(boolean val) {
     	setInitSuccess(val);
-    	session.onSecurityInitialized();
+		if (session != null)
+			session.onSecurityInitialized();

```

Comment:
A session needs to be established prior to the initialization (we need an RAI response), please see the code snippet below:
https://github.com/smartdevicelink/sdl_android/blob/41a6698a2cbd7b3428fc3be55efe30fc21c3965a/sdl_android/src/main/java/com/smartdevicelink/proxy/SdlProxyBase.java#L1781-L1823